### PR TITLE
Final Frontier: Added version_min/max to indicate compatibility with 1.0.4

### PR DESCRIPTION
**Pull request by MattMaginniss**

### CODE REVIEW

File: NetKAN/FinalFrontier.netkan [hunk 1]
```diff
@@ -13,4 +13,14 @@
             "filter": [ "000_Toolbar", "Source" ]
        }
     ]
```
Comment (SkyMarshal) [inline NetKAN/FinalFrontier.netkan]: Looks to be missing a comma.

**Comment by MattMaginniss** [comment]
#rebuild

**Comment by SkyMarshal** [comment]
You may want to squash the commits.

**Comment by pjf** [comment]
Thank you!!

**Pull request merged**